Pull-RQ for chapter 3

TatianaPivovar opened the pull request:
We start!

TatianaPivovar committed:
Start the drill in ch.3

the drill 3.1 solved

TatianaPivovar committed:
drill 3.2

solved drill 3.2

TatianaPivovar committed:
drill 3.3

the drill 3.3 solved

TatianaPivovar committed:
drill 3.4

the drill 4.4 solved

TatianaPivovar committed:
drill 3.5

the drill 3.5 solved

TatianaPivovar committed:
drill 3.6

the drill 3.6 solved

TatianaPivovar committed:
drill 3.7

the drill 3.7 solved

TatianaPivovar committed:
exercises 1-11

all exercises solved

TatianaPivovar committed:
Build script for windows and MSVS

I took this script from www.stroustroup.com and just add
%VS140COMNTOOLS%

HappyJonny90 commented on Chapter03/Drill3.cpp:
```diff
@@ -0,0 +1,10 @@
+
+int main()
+{
+    std::cout << "Enter the name of the person you want to write to:\n";
```
Why do you use C-style '\n'-symbol instead of C++-style std::endl?

TatianaPivovar replied:
Осталось после первой версии :) Но следующим коммитом это было исправлено.

HappyJonny90 commented on Chapter03/Drill3.cpp:
```diff
@@ -0,0 +1,10 @@
+
+int main()
+{
+    std::cout << "Enter the name of the person you want to write to:\n";
```
Why do you use C-style '\n'-symbol instead of C++-style std::endl?

HappyJonny90 replied:
Why do you use C-style '\n'-symbol instead of C++-style std::endl?

TatianaPivovar committed:
Add ignore list and attributes for git

TatianaPivovar committed:
Fixed bug with infinite loop

Added prompt
Fixed bug with infinite loop
'Magic' constant replaced with a meaninful function call

TatianaPivovar committed:
Add newline to ends of files

GitHub shows "No newline at and of file" as error
